src/table/sticky-columns/use-sticky-columns.ts

```diff
@@ -208,13 +208,18 @@ export function useStickyCellStyles({
         }
       };
 
-      const unsubscribe = stickyColumns.store.subscribe(selector, (newState, prevState) =>
-        updateCellStyles(selector(newState), selector(prevState))
-      );
+      // Initial render
+      const update = () => {
+        stickyColumns.store.subscribe(selector, (newState, prevState) => {
+          updateCellStyles(selector(newState), selector(prevState));
+        });
+      };
+
+      update();
+
       return () => {
-        unsubscribe();
-        // Force the cleanup
-        updateCellStyles(null, { padLeft: false, lastLeft: false, lastRight: false, offset: {} });
+        // Unsubscribe / force cleanup
+        update();
```

Comment:
I don't quite get this line. The `update()` call triggers subscribe, not unsubscribe - isn't it?

Reply:
That makes sense. Not only does it look cleaner, by having the conditional checks it should also solve the `StrictMode` issue. I'll try that approach.